### PR TITLE
feat(datepicker): datepicker doc and style migration

### PR DESCRIPTION
**Pull request by iropolo**
## Datepicker component documentation and style migration

[DLT-1132
](https://dialpad.atlassian.net/jira/software/c/projects/DLT/boards/548?modal=detail&selectedIssue=DLT-1132&assignee=6368be5e3867a77cb8d64d52)

### CODE REVIEW

File: lib/build/less/components/datepicker.less [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    border: none;
+    border-radius: calc(var(--dt-size-650) + var(--dt-size-200));
+    cursor: pointer;
+
+    &--disabled {
+      color: var(--black-300);
+    }
+
+    &--selected {
+     color: #fff;
```
Comment (braddialpad) [inline lib/build/less/components/datepicker.less]: hmm should this change for dark mode? if so `--dt-color-foreground-primary`

File: lib/build/less/components/datepicker.less [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+     background: #7c52ff;
```
Comment (braddialpad) [inline lib/build/less/components/datepicker.less]: dt color can be used for this?

File: lib/build/less/components/datepicker.less [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    > div {
+        display: inline-flex;
+      }
+
+    button {
+      display: inline-flex;
+      align-items: center;
+      padding: 0;
+      color: var(--primary-color);
```
Comment (braddialpad) [inline lib/build/less/components/datepicker.less]: hmm think we shouldn't use --primary-color for anything purple 400 or 500 instead?

File: lib/build/less/components/datepicker.less (hunk 2 above)
Comment (iropolo) [reply lib/build/less/components/datepicker.less]: replaced it with `var(--dt-color-border-brand)`

File: lib/build/less/components/datepicker.less (hunk 3 above)
Comment (iropolo) [reply lib/build/less/components/datepicker.less]: replaced it with `var(--purple-400)`

**Comment by iropolo** [comment]
Color migration following guide [Migration Guide Dialtone 8](https://github.com/dialpad/dialtone/blob/version8/migration_guide/Dialtone_8.md)

**Review by braddialpad — COMMENTED** [review]
Hmm Vue API and example not showing.. Might just need to do the same thing you did on emoji picker.

**Comment by iropolo** [comment]
> Hmm Vue API and example not showing.. Might just need to do the same thing you did on emoji picker.

Yes your right, I didnt push package updates because I did it in emoji-picker PR.
Im guessing if we merge emoji-picker this will work, I dont see it necessary to update it here also.

Lemme merge emoji-picker doc then will update this PR with it.

image

**Comment by iropolo** [comment]
Now It works in the preview 👍🏼

**Review by braddialpad — APPROVED** [review]

**Pull request merged**